### PR TITLE
Improve grammar in `README` file.

### PR DESCRIPTION
**Pull request by savagemike**

### CODE REVIEW

**Comment by rmariuzzo** [comment]
There are still conflicts, can you fix them?

**Comment by savagemike** [comment]
@rmariuzzo sorry about that! Should be all fixed now.

File: README.md [hunk 1]
```diff
@@ -2,13 +2,16 @@
...
+
 **Local open source communities** is an initiative to gather all possible public data about projects from the Dominican Republic in GitHub using the [GraphQL API](https://developer.github.com/v4/). Build and serve a public website showing all the local efforts that Dominicans are contributing toward open source.
 
+
 ## Want to join?
 
 There are different ways to participate in this local effort:
 
- - You can include your GitHub users by adding the term “dominican” to your profile's location(dominicana is also accepted).
+p
```
Comment (rmariuzzo) [inline README.md]: @savagemike I think that `p` should be removed. :)

Comment (savagemike) [reply README.md]: I did a new pull request, should be fixed. Let me know!

**Comment by rmariuzzo** [comment]
Thank you @savagemike !

**Pull request merged**